.github/workflows/actions.yml

```diff
@@ -1,17 +1,17 @@
 name: linux-build-dist
-on: [push, pull_request]
+on: [push, pull_request, workflow_dispatch]
```

Comment:
I wonder what `workflow_dispatch` do here? It seems if I submit a PR, the workflow is now triggered twice, once for the "pull_request" and once for the "workflow_dispatch"

Reply:
It should just mean that a button appears in the actions tabs that mean you can run the action 'manually'. It shouldn't do anything automatically.

Reply:
> a button appears in the actions tabs that mean you can run the action 'manually'

Ok, thanks!